src/editor/init.js

```diff
@@ -7,15 +7,8 @@
 var _initialized = false;
 var _defaultTheme = "";
 
-function addStylesheet(path) {
-    var link = document.createElement("link");
-    link.href = path;
-    link.type = "text/css";
-    link.rel = "stylesheet";
-    link.media = "screen,print";
-
-    document.getElementsByTagName("head")[0].appendChild(link);
-    return link;
+function addStylesheet(sheet){
+	document.getElementById('pagestyle').setAttribute('href', sheet);
```

Comment:
@danieleds the behavior is altered but it still has the same effect- just better. It used to create a new link element  every time the theme was changed. By appending it to the document it would overwrite the previously created link element.

What @Teklad did was to create one named link element and just set its source to whichever theme.css was selected.